src/pages/podcasts/[slug].tsx

```diff
@@ -46,7 +25,13 @@ const MorePodcasts: FunctionComponent<MorePodcastProps> = ({podcasts}) => (
       </h3>
       <ul className="mb-10 justify-items-center grid gap-4 grid-cols-1 sm:grid-cols-2 md:grid-cols-3">
         {podcasts.map((podcast: PodcastResource) => (
-          <PodcastCard podcast={podcast} key={podcast.id} />
+          <PodcastCard
```

Comment:
Should this just take a `PodcastResource` as a prop instead of breaking it apart like this in the parent? Then the card can make its own life choices about what it uses.

Reply:
I'm cool either way on this 👍 

I think because of `TypeScript`, it's more forgiving because that will enforce what we're up to.

Happy to revert it back. The only downside is that it's not overly informative if someone views the code in the Storybook. However, that's easily mitigated by documenting things if needed in the MDX.